File: teoricos/tt2/exercicios_revisão/Questao1e2. java/questao1.java
class questao1 {
    
    public static boolean estaContido(int array[],int num)
    {
        boolean contem = false;
        
        int i = 0;
        for(; i < array.length; i++)        
        {
            if(array[i] == num) 
            {             
                contem = true;
                i = array.length;
            }
        } 

        return contem;
    }

    public static void main (String[] args)
    {
        int[] array = {1,2,3,4,5};
        int num = 0;

        boolean contem = estaContido(array, num);
        System.out.println(contem);
    }
}

File: teoricos/tt2/labs/Questao1.cpp
#include <iostream>
#include <stdbool.h>

using namespace std;

// Faça um método que receba um array de inteiros e um
// número inteiro x e retorne um valor booleano informando se o
// elemento x está contido no array

bool estaContido(int array[], int num)
{
    bool achou = false;
    
    int i = 0;

    while(!achou && i < sizeof(array))
    {
        if(num == array[i]) achou = true;
        i++;
    }

    return achou;
}

int main()
{
    int array[5] = {1,2,3,4,5};
    int num = 0;    
    bool flag = estaContido(array, num);

    cout << "\n" << flag << "\n";

    return 0;
}
